modules/google-cloud/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+    `java-library`
+    id("dev.clojurephant.clojure")
+    `maven-publish`
+    signing
+}
+
+publishing {
+    publications.create("maven", MavenPublication::class) {
+        pom {
+            name.set("XTDB Google Cloud")
+            description.set("XTDB Google Cloud")
+        }
+    }
+}
+
+dependencies {
+    api(project(":api"))
+    api(project(":core"))
+
+    api("com.google.cloud", "google-cloud-storage", "2.23.0")
+    api("com.google.guava", "guava", "31.0.1-jre")
```

Comment:
you may not need guava in Gradle, if it's just for dependency resolution - Gradle tends to be much better at resolving dependencies

Reply:
:eyes: That's useful to know - what would be a good way of confirming that? Attempting to run the tests without it, perhaps?